erpnext/healthcare/doctype/lab_test/lab_test.py

```diff
@@ -10,26 +10,27 @@
 
 class LabTest(Document):
 	def on_submit(self):
-		frappe.db.set_value(self.doctype,self.name,"submitted_date", getdate())
+		frappe.db.set_value(self.doctype,self.name,'submitted_date', getdate())
```

Comment:
Use `db_set` method in class methods

```suggestion
		self.db_set('submitted_date', getdate())
```